electrode-ota-server-model-acquisition/src/acquisition.js

```diff
@@ -27,7 +27,7 @@ export default (options, dao, weighted, _download, manifest, logger) => {
          */
         updateCheck(params) {
 
-            missingParameter(params.deploymentKey, `Deployment key missing`);
+            //missingParameter(params.deploymentKey, `Deployment key missing`);
             missingParameter(params.appVersion, `appVersion missing`);
```

Comment:
Let's move appVersion to the other location as well.

Reply:
moved both up to the route handler, as of not this method is invoked from route handler.